Source/Scene/PointCloudEyeDomeLighting.js

```diff
@@ -0,0 +1,400 @@
+define([
+        '../Core/Cartesian2',
+        '../Core/Math',
+        '../Core/clone',
+        '../Core/Color',
+        '../Core/combine',
+        '../Core/ComponentDatatype',
+        '../Core/defined',
+        '../Core/destroyObject',
+        '../Core/FeatureDetection',
+        '../Core/Geometry',
+        '../Core/GeometryAttribute',
+        '../Core/PixelFormat',
+        '../Core/PrimitiveType',
+        '../Renderer/BufferUsage',
+        '../Renderer/ClearCommand',
+        '../Renderer/DrawCommand',
+        '../Renderer/Framebuffer',
+        '../Renderer/Pass',
+        '../Renderer/PixelDatatype',
+        '../Renderer/RenderState',
+        '../Renderer/Sampler',
+        '../Renderer/ShaderSource',
+        '../Renderer/ShaderProgram',
+        '../Renderer/Texture',
+        '../Renderer/TextureMagnificationFilter',
+        '../Renderer/TextureMinificationFilter',
+        '../Renderer/TextureWrap',
+        '../Renderer/VertexArray',
+        '../Scene/BlendEquation',
+        '../Scene/BlendFunction',
+        '../Scene/BlendingState',
+        '../Scene/StencilFunction',
+        '../Scene/StencilOperation',
+        '../Shaders/PostProcessFilters/PointCloudEyeDomeLighting'
+    ], function(
+        Cartesian2,
+        CesiumMath,
+        clone,
+        Color,
+        combine,
+        ComponentDatatype,
+        defined,
+        destroyObject,
+        FeatureDetection,
+        Geometry,
+        GeometryAttribute,
+        PixelFormat,
+        PrimitiveType,
+        BufferUsage,
+        ClearCommand,
+        DrawCommand,
+        Framebuffer,
+        Pass,
+        PixelDatatype,
+        RenderState,
+        Sampler,
+        ShaderSource,
+        ShaderProgram,
+        Texture,
+        TextureMagnificationFilter,
+        TextureMinificationFilter,
+        TextureWrap,
+        VertexArray,
+        BlendEquation,
+        BlendFunction,
+        BlendingState,
+        StencilFunction,
+        StencilOperation,
+        PointCloudEyeDomeLightingShader
+    ) {
+    'use strict';
+
+    /**
+     * Eye dome lighting.
+     * Requires support for EXT_frag_depth, OES_texture_float, and WEBGL_draw_buffers extensions in WebGL 1.0.
```

Comment:
This should be documented in the public API - not just here.

What happens when these are not supported?  Crash?  Ignored?  Console warning?

Reply:
It just gets ignored. I'll add a note in `PointAttenuationOptions`/`PointShading`.

Reply:
OK, please add a `isSupported` function to the right place if there isn't already one.

For example, see https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/ClassificationPrimitive.js#L346